src/test/java/com/example/housingmanagement/api/services/HouseServiceTest.java

```diff
@@ -0,0 +1,223 @@
+    void test5() {
+        //given
+        Mockito.when(houseRepositoryMock.findById(0)).thenReturn(Optional.of(sampleHouseInternalEntity));
+
+        //when
+        Optional<HouseInternalEntity> sampleOptional = houseService.findById(0);
+
+        //then
+        Assertions.assertEquals(sampleOptional, Optional.of(sampleHouseInternalEntity));
+    }
+
+    @Test
+    @DisplayName("Should call a method deleteByHouseNumber from houseRepository when deleteHouseFromDatabase is called")
+    void test6() {
+        //given
+        HouseRequest sampleRequest = sampleHouseRequest;
+
+        //when
+        houseService.deleteHouseFromDatabase(sampleRequest);
+
+        //then
+        Mockito.verify(houseRepositoryMock).deleteByHouseNumber(sampleHouseRequest.getHouseNumber());
+    }
+
+    @Test
+    @DisplayName("Should return a current capacity of HouseInternalEntity")
+    void test7() {
+        //given
+        Mockito.when(houseRepositoryMock
+                .findByHouseNumber(sampleHouseRequest.getHouseNumber())).thenReturn(sampleHouseInternalEntity);
+
+        //when
+        int capacity = houseService.houseCurrentCapacity(sampleHouseRequest);
+
+        //then
+        Mockito.verify(houseRepositoryMock).findByHouseNumber(sampleHouseRequest.getHouseNumber());
+        assertEquals(sampleHouseInternalEntity.getCurrentCapacity(), capacity);
+    }
+    @Test
+    @DisplayName("Should return true if HouseInternalEntity exists in db")
+    void test8(){
+        //given
+        Mockito.when(houseRepositoryMock.existsByHouseNumber(sampleHouseRequest.getHouseNumber())).thenReturn(true);
+
+        //when
+        boolean exists = houseService.existsByHouse(sampleHouseRequest);
+
+        //then
+        Assertions.assertTrue(exists);
+    }
+    @Test
+    @DisplayName("Should return false if HouseInternalEntity does not exist in db")
+    void test9(){
+        //given
+        Mockito.when(houseRepositoryMock.existsByHouseNumber(sampleHouseRequest.getHouseNumber())).thenReturn(false);
+
+        //when
+        boolean exists = houseService.existsByHouse(sampleHouseRequest);
+
+        //then
+        Assertions.assertFalse(exists);
+    }
+    @Test
+    @DisplayName("Should return true if HouseInternalEntity has spare capacity")
+    void test10(){
+        //given
+        Mockito.when(houseRepositoryMock
+                .findByHouseNumber(sampleHouseRequest.getHouseNumber())).thenReturn(sampleHouseInternalEntity);
+
+        //when
+        boolean hasSpareCapacity = houseService.houseHasSpareCapacity(sampleHouseRequest);
+
+        //then
+        Assertions.assertTrue(hasSpareCapacity);
+    }
+
+    @Test
+    @DisplayName("Should increase HouseInternalEntity capacity by one based on HouseRequest")
+    void test11() {
+        //given
+        HouseInternalEntity foundHouseInternalEntity = new HouseInternalEntity(1, LocalDateTime.now(), LocalDateTime.now(),
```

Comment:
LocalDateTime.now() should be assigned to the variable 'now' or something like that.
And then use 'now' instead of calling two times 'LocalDateTime.now()'